test/NuGet.Core.FuncTests/NuGet.Commands.FuncTest/RestoreCommandTests.cs

```diff
@@ -3560,11 +3561,19 @@ await SimpleTestPackageUtility.CreateFolderFeedV3Async(
             var restoreGraph = result.RestoreGraphs.ElementAt(0);
             Assert.Equal(0, restoreGraph.Unresolved.Count);
             //packageA should be installed from source2
-            string packageASource = restoreGraph.Install.ElementAt(0).Provider.Source.Name;
-            Assert.Equal(packageSource2, packageASource);
-            //packageB should be installed from source
-            string packageBSource = restoreGraph.Install.ElementAt(1).Provider.Source.Name;
-            Assert.Equal(pathContext.PackageSource, packageBSource);
+            restoreGraph.Install.Count.Should().Be(2);
```

Comment:
This test needed to be updated because the list of packages to install has non-deterministic ordering.  It now checks that there are the correct number of packages to install and that each package is coming from the correct source rather than checking if `list[0]` and `list[1]` are.